### ISSUE TITLE
API Errors

### ISSUE THREAD

**Issue by lots0logs** [bcedu/VGrive#52]
```
{
"error": {
"errors": [
{
"domain": "global",
"reason": "invalid",
"message": "Invalid Value",
"locationType": "parameter",
"location": "q"
}
],
"code": 400,
"message": "Invalid Value"
}
}

trashed = False and name = 'Hooks & Filters Wish List' and '1ohAJZ9mrHErsRP-ny9Li4mCoLIIyTiLH' in parents
```
```
{
"error": {
"errors": [
{
"domain": "global",
"reason": "invalid",
"message": "Invalid Value",
"locationType": "parameter",
"location": "q"
}
],
"code": 400,
"message": "Invalid Value"
}
}

trashed = False and name = 'Divi X - Proposal - 1 - Render & Save in JS -> Serve rendered HTML' and '1XhvKUq1-PECcAPwXKvpU0n4XJwWU8F4W' in parents
```

**Comment by bcedu** [comment]
The proboem is the char '&'.
Im working on it

**Comment by bcedu** [comment]
Closed with: ee45f774841824ce6e5fc608905a64b1bf90fed0
